plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/GlueHiveMetastore.java

```diff
@@ -695,7 +696,11 @@ private TableInput convertGlueTableToTableInput(com.amazonaws.services.glue.mode
     @Override
     public void commentTable(String databaseName, String tableName, Optional<String> comment)
```

Comment:
Please add a test in `TestHiveGlueMetastore` to ensure test coverage for this new functionality.